interfaces/src/defs.rs

```diff
@@ -1,14 +1,16 @@
+#[derive(Clone, Debug)]
```

Comment:
I added several of these too recently. if it causes conflicts could you re-add them, sorry